src/referral-program/referral-program.controller.spec.ts

```
@@ -0,0 +1,58 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { ReferralProgramController } from './referral-program.controller';
+import { ReferralProgramService } from './referral-program.service';
+import { ClaimReferralDto } from './dto/claim-referral.dto';
+
+describe('ReferralProgramController', () => {
+  let controller: ReferralProgramController;
+  let service: ReferralProgramService;
```

ESLint

[error] 'service' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)